packages/vega-scenegraph/src/SVGRenderer.js

```diff
@@ -529,6 +532,9 @@ prototype._update = function(mdef, el, item) {
   // apply svg attributes
   mdef.attr(emit, item, this);
 
+  // apply aria-specific properties
+  ariaItemAttributes(emit, item, this);
```

Comment:
This function only takes two arguments, no?

Reply:
This was fixed in 5.11.1